src/harness/sas_interface.py

```diff
@@ -374,6 +374,21 @@ def TriggerPpaCreation(self, request):
     """
     pass
 
+  def TriggerDailyActivitiesImmediately(self):
+    """SAS admin interface to trigger daily activities immediately which will
+    execute the activities mentioned in the SAS admin agreement
+    """
+    pass
+
+  def GetDailyActivitiesStatus(self):
+    """SAS admin interface to get the daily activities status
+    Returns:
+      A dictionary with single key-value pair where the key is "status" and the
+      value is a string of permitted enumerations specified in SAS Under Testing
```

Comment:
See my comment on the testing API doc re: enum vs. boolean. I think we can simplify our lives :)